www/js/recommendations.js

```diff
@@ -6,15 +6,19 @@ angular.module('emission.main.recommendations',['emission.services', 'emission.p
                 'emission.splash.referral', 'angularLocalStorage',
                 'ng-walkthrough', 'nzTour'])
 
+/*
+Ask Shankari about this Wednesday
+.service('CommHelper', function($http) {
```

Comment:
`CommHelper` is already defined in `services.js`. Unsure why you think you need to redefine it here?